File: StarWars/APICORE/Entities/Product.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace APICORE.Entities
{
    [Table("Product")]
    public class Product
    {
        [Key]
        public int Id { get; set; }
        public string Title { get; set; }
        public int Price { get; set; }
        public string Zipcode { get; set; }
        public string Seller { get; set; }
        public string ThumbnailHd { get; set; }

        [Column("ProductDate")]
        public string Date { get; set; }
    }
}

File: models/product.js
"use strict"

const mongoose = require('../bin/db');
const Schema = mongoose.Schema;

const schema = new Schema({
    title: {
        type:String
    },
    price: {
        type: Number
    },
    zipcode: {
        type: String
    },
    seller: {
        type: String
    },
    thumbnailHd: {
        type: String
    },
    date: { 
        type: String
    }
});

// Duplicate the ID field.
schema.virtual('product_id').get(function(){
    return this._id.toHexString();
});

module.exports = mongoose.model("Product", schema);